guides/roles/driver.md

```diff
@@ -0,0 +1,36 @@
+# Driving Guide
+
+## Required Reading
+
+Drivers, Navigators, and Gunners are expected to be familiar with [Convoy Tactics](guides/tactics/convoy.md). It is recommended to read this document first.
+
+## Terminology
+
+*Hard Dismount* Everyone dismounts from the vehicle  
+*Soft Dismount* Driver and Gunners stay mounted, everyone else dismounts  
+*VRP - Vehicle Rally Point* If a vehicle should be left behind during an attack, it should be left at an agreed-upon location that is out of sight.
+
+## Vehicle Types
+
+### Unarmored Transport
+Vehicles that are used for high speed, high capacity, or long-range transportation. They should never be expected to be used in combat and are rarely equipped with any defenses.
+
+*Examples* Prowler, HEMTT, Civilian Vehicles
+
+### Armored Transport
+Vehicles that are used for high mobility light combat transportation. Usually equipped with some mounted weapon for defense.
+
+*Examples* HMMWV, Hunter, Strider
```

Comment:
The last two are all [MRAPs](https://en.wikipedia.org/wiki/Oshkosh_M-ATV) though...

In the game the Fennek is meant to act as an MRAP aswell.

Reply:
Are we using ACE vehicle names or ARMA vehicle names?

Reply:
Going to use ACE names